packages/io-devices/src/index.ts

```diff
@@ -0,0 +1,124 @@
+import { WasmFs } from "../../wasmfs/src/index";
+
+// Define our file paths
+const FRAME_BUFFER = "/dev/wasmerfb0";
+const WINDOW_SIZE = "/sys/class/graphics/wasmerfb0/virtual_size";
+const BUFFER_INDEX_DISPLAY =
+  "/sys/class/graphics/wasmerfb0/buffer_index_display";
+const INPUT = "/dev/input";
+
+export default class IoDevicesDefault {
+  wasmFs: WasmFs;
+  fdFrameBuffer: number;
+  fdWindowSize: number;
+  fdBufferIndexDisplay: number;
+  fdInput: number;
+
+  windowSizeCallback: Function;
+  bufferIndexDisplayCallback: Function;
+  inputCallback: Function;
+
+  constructor(wasmFs: WasmFs) {
+    this.wasmFs = wasmFs;
+
+    // Add our files to the wasmFs
+    const wasmFsJSON = this.wasmFs.toJSON();
+    wasmFsJSON[FRAME_BUFFER] = "";
+    wasmFsJSON[WINDOW_SIZE] = "";
+    wasmFsJSON[BUFFER_INDEX_DISPLAY] = "";
+    wasmFsJSON[INPUT] = "";
+    this.wasmFs.fromJSON(wasmFsJSON);
```

Comment:
This lines are creating empty files for the corresponding handles, but is done in a way that is not very optimal.

You can do this instead:

```suggestion
this.wasmFs.volume.writeFileSync(FRAME_BUFFER, "");
this.wasmFs.volume.writeFileSync(WINDOW_SIZE, "");
this.wasmFs.volume.writeFileSync(BUFFER_INDEX_DISPLAY, "");
this.wasmFs.volume.writeFileSync(INPUT, "");
```

Which is simpler and a bit more optimal :)

Reply:
Done 😄

Reply:
Oh wait! The reason why I did it this was, was because this will create the directories, as well as the files all at once 😯 So We could do it where we create all the directories using the node API, but I feel like that would be a lot more code?

Reply:
Then, we can create the dirs. That's a much better way since is explicit and a bit more performant as doesn't require serializing and deserializing data constantly.

```js
    this.wasmFs.volume.mkdirSync("/sys/class/graphics/wasmerfb0", { recursive: true });
    this.wasmFs.volume.writeFileSync(FRAME_BUFFER, "");
    this.wasmFs.volume.writeFileSync(WINDOW_SIZE, "");
    this.wasmFs.volume.writeFileSync(BUFFER_INDEX_DISPLAY, "");
    this.wasmFs.volume.writeFileSync(INPUT, "");
```

Reply:
Cool, I'll go ahead and do that 😄